src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql

```diff
@@ -83,7 +83,26 @@ VALUES
     ('02438278-ab93-4689-b123-7ac4e78f59fe', 'smsWebEmail_1000'    , 'responses per site'           , '1000 SMS/web/email responses per site'),
     ('67e1c174-8443-4883-b51f-43c297fa9c08', 'smsWebEmail_1500'    , 'responses per site'           , '1500 SMS/web/email responses per site'),
     ('1a36f980-76a4-4b3d-b36d-46afc6655124', 'dataMigration'       , 'data migrations'              , 'per data migration'),
-    ('8413d5e1-2995-4885-9ffa-ec961d82aa6d', 'sharedService'       , 'shared services'              , 'per shared service');
+    ('8413d5e1-2995-4885-9ffa-ec961d82aa6d', 'sharedService'       , 'shared services'              , 'per shared service'),
+    ('f24afc45-971d-41fa-9beb-6f577e987c6d', 'patients_1-20k'      , 'patients'                     , 'per patient for 1-19,999 patients'),
```

Comment:
The hyphen-minus character should be an en dash, e.g. `'per patient for 1–19,999 patients'`.

Reply:
Updated description to use hyphen-minus character.